Assets/ColyseusClient.cs

```diff
@@ -151,11 +149,4 @@ void OnChangeFallback (PatchObject change)
 //		Debug.Log (change.path);
 //		Debug.Log (change.value);
 	}
-
-	void OnApplicationQuit()
-	{
-		// Make sure client will disconnect from the server
-		room.Leave ();
-		client.Close ();
-	}
```

Comment:
The problem I see here is when you hit the "play" button on the editor, and then stop the game, the connection is not fully closed. The server continues having the client reference there.